packages/select/src/FilterMultiSelect/components/Trigger/TriggerButtonBase/TriggerButtonBase.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react"
+import chevronDown from "@kaizen/component-library/icons/chevron-down.icon.svg"
+import chevronUp from "@kaizen/component-library/icons/chevron-up.icon.svg"
+import { Icon } from "@kaizen/component-library"
+
+import classNames from "classnames"
+import { useMenuTriggerContext } from "../../../provider/MenuTriggerProvider"
+import styles from "./TriggerButtonBase.scss"
+
+export type TriggerButtonBaseProps = {
+  children: React.ReactNode
+  ClassNameOverriden?: string
```

Comment:
Is this meant to be `classNameOverride` like our other components, or is there a reason why you did a capital first letter and slightly different name?

Reply:
yes it is meant to be classNameOverride, thank you for picking it up!